.github/workflows/l10n-extract.yml

```diff
@@ -0,0 +1,46 @@
+name: String extraction test
+on:
+  pull_request:
+jobs:
+  extract:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Linux packages
+        run: |
+          sudo apt update
+          sudo apt install gettext -y
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Install global npm packages
+        run: |
+          npm install -g grunt-cli
+      - name: Clone l10n repository
+        uses: actions/checkout@v3
+        with:
+          repository: mozilla/fxa-content-server-l10n
+          path: "fxa-l10n"
+      - name: Clone FxA code repository
+        uses: actions/checkout@v3
+        with:
+          repository: "bcolsson/fxa"
```

Comment:
Is this your fork? Or should this be `"mozilla/fxa"`?

Reply:
@pdehaan Yes, you're right. Fixed!